README.md

```diff
@@ -1,110 +1,161 @@
-# Digital Identity Core Front
+# ipv-core-front
 
-`di-ipv-core-front`
+This repository contains the source code for the frontend user interface of the GOV.UK One Login Identity Proofing and Verification (IPV) system. The IPV Core frontend serves as the first user-facing screen within the identity proving journey.
 
-This is the home for the core front end user interface for the Identity Proofing and Verification (IPV) system within the GDS digital identity platform, GOV.UK Sign In. This is the first user facing screen within the Identity Proving journey. Other repositories are used for core backend services or credential issuers.
+## Related repositories
 
-# Installation
+ipv-core-front works with the following IPV Core repositories:
+
+* [ipv-core-back](https://github.com/govuk-one-login/ipv-core-back) - backend code
+* [ipv-core-tests](https://github.com/govuk-one-login/ipv-core-tests) - feature tests
+* [ipv-core-common-infra](https://github.com/govuk-one-login/ipv-core-common-infra) - utilities for automating IPV Core ancillary services
+* [ipv-stubs](https://github.com/govuk-one-login/ipv-stubs) - stubs for IPV Core dependencies (credential issuers)
+
+# Developing ipv-core-front
+
+This guide explains how to:
+
+* clone the repo and install the dependencies
+* run ipv-core-front locally
+* use pre-commit to verify your commits
+
+## Installing ipv-core-front
+
+1. Clone this repository to your local machine:
+
+```
+https://github.com/govuk-one-login/ipv-core-front.git
+```
+
+2. Run the following command to install the project dependencies:
 
-Clone this repository and then run
 ```bash
 npm install
 ```
 
-## Environment Variables
+### Environment variables
+
+The following environment variables are used in the project:
+
+| Variable Name             | Description                                              | Default Value  |
+|---------------------------|----------------------------------------------------------|----------------|
+| `BASE_URL`                | Externally accessible base URL of the webserver. Used to generate callback URLs for the credential issuer OAuth (Open Authorization) flows. | -         |
+| `API_BASE_URL`            | Specifies the base host of the backend API. It is used by the application to make requests to the backend services.                         | -         |
+| `PORT`                    | Default port to run the webserver on.                    | `3000`         |
+| `EXTERNAL_WEBSITE_HOST`   | Sets the default host used by the application.                                                         |   `http://localhost:8080`                   |
+| `NODE_ENV`                | Specifies the environment where the application will run, for example `local`.                                                   |             -        |
+| `SESSION_SECRET`          | The secret key required for encrypting and decrypting session data.                                                       |             -         |
```

Comment:
Could we add:
- `ENABLE_PREVIEW` to enable the `dev/all-templates` route to preview individual pages - default value `development`
- `CONTACT_URL` URL of the One Login contact form `https://home.account.gov.uk/contact-gov-uk-one-login`

and remove:
- `BASE_URL` (not used anymore)